packages/terra-form-fieldset/docs/README.md

```diff
@@ -0,0 +1,37 @@
+# Terra Form Fieldset
+
+Generic form fieldset component which handles the layout of a standard form fieldset including errors, help text, legend, value and widget placement.
+
+## Getting Started
+
+- Install with [npmjs](https://www.npmjs.com):
+  - `npm install terra-form-fieldset`
+
+## Usage
+
+```jsx
+import React from 'react';
+import Fieldset from 'terra-form/lib/Fieldset';
+
+<Fieldset
+  legend="Do you have any Children?"
+  legendAttrs={{ className: 'healtheintent-application' }}
+  error="This field is required"
+  help="Families are eligible for family package plans"
+  required
+  isInline
+ >
+   {this.props.children}
+</Fieldset>
+```
+
+## Component Features
+
+<!-- Uncomment supported features.
```

Comment:
You can uncomment any supported features this component has. This helps users better understand what the component offers.